packages/js/docs/tsc-examples.md

```diff
@@ -0,0 +1,75 @@
+## Examples
+
+{% tabs %}
+{% tab label="Using TypeScript Transformer Plugins" %}
+
+`@nrwl/js:tsc` can run the [TypeScript Transformers](https://github.com/madou/typescript-transformer-handbook) by using the `transformers` option.
+
+```json
```

Comment:
If needed you can pass the name of the file in a code fence with the `{% fileName="path/to/my/file.ext" %}`.
More details here https://github.com/nrwl/nx/blob/master/docs/README.md#code.

Reply:
Thanks!